src/main/java/controller/BookController.java

```diff
@@ -30,32 +31,37 @@ public class BookController {
     private final BookMapper bookMapper;
 
     @GetMapping
+    @PreAuthorize("hasRole('USER')")
     @Operation(summary = "Get all books", description = "Get a list of all available books")
     public List<BookDto> findAll(Pageable pageable) {
         return bookService.findAll(pageable);
     }
 
     @GetMapping("/{id}")
+    @PreAuthorize("hasRole('USER')")
```

Comment:
It would be great to allow this endpoint for admin =)
```suggestion
    @PreAuthorize("hasAnyRole('USER', 'ADMIN')")
```